pegasus/sites.v3/hourofcode.com/public/promote/index.md.partial

```diff
@@ -6,9 +6,9 @@ nav: promote_nav
 
 {{ signup_button }}
 
-# Get your community involved in the Hour of Code  
+# Get your community involved in the Hour of Code
```

Comment:
Is this page translated at all?

Reply:
I'm also now realizing all the changes in this PR are whitespace

Reply:
I'm going to delete and redirect this page and all of it's translations in this ticket: [ACQ-2414](https://codedotorg.atlassian.net/browse/ACQ-2414)